RetailSuiteAccounting/UserInterface/01f_BookReceipts.md

```diff
@@ -0,0 +1,124 @@
+
+- *Receipt 2*  
+Enter any other appropriate reference value, such as order number.
+
+- *Date*  
+Enter the posting date. The valid date format is DDMMYY.
+
+- *Account*   
+Enter the appropriate account number, for example a personal account number, that is, customer or supplier number. A valid range for personal account numbers (10000-69999 for customers and 70000-99999 for suppliers) is predefined in the system.
+
+  > [Info] Customer and supplier personal accounts can be created, edited and deleted. For detailed information, see [Manage the customer/supplier](../Integration/05_ManageCustomerSupplier.md).
+
+- *COST1*  
+Enter the appropriate value.
+
+  > [Info] This is only applicable if at least one cost center has been set up. For detailed information, see [COST ACCOUNTING](./02e_CostAccounting.md).
+
+- *COST2*  
+Enter the appropriate value.
+
+  > [Info] This is only applicable if at least one cost unit has been set up. For detailed information, see [COST ACCOUNTING](./02e_CostAccounting.md).
+
+- *Posting text*  
+Enter an appropriate posting text. The posting text can contain a combination of letters and numbers.
+
+
+When creating a manual posting, the account/contra account details, as well as their current balances, are displayed under the input line.
+
+- *Contra account*  
+This field displays the selected contra account. This field is read-only.
+
+- *Account*  
+This field displays the selected account. This field is read-only.
+
+- *Balance (amount)H/S (currency)*  
+This field displays the current balance of the corresponding account. H stands for credit. S stands for debit. This field is read-only.
+
+
+
+- [POST]    
+Click this button to post the transaction entered in the input line.
+
+- [CANCEL]  
+Click this button to cancel a selected posting in the *Postings* tab. This button is not relevant for this tab. For detailed information, see [Cancel a posting](../Operation/05_CancelBooking.md).
+
+- [DELETE]    
+Click this button to delete a selected posting in the *Postings* tab. This button is not relevant for this tab. For detailed information, see [Delete a single posting](../Operation/06_DeleteBookings.md#delete-a-single-posting).
+
+- [CLEAR]    
+Click this button to clear all input line fields.
+
+- [SPLIT]    
+Click this button to split a manual posting. The *Split posting* window is displayed, see [Split posting](./01a_Bookings.md#split-posting). For detailed information about splitting a posting, see [Split a posting](../Operation/09_SplitBooking.md).
+
+- [ASSIGN RECEIPT]  
+Click this button to assign a receipt to a selected posting in the *POSTINGS* tab. This button is not relevant for this tab. For detailed information, see [Retrieve a receipt](../Operation/10_ManageReceipts.md#retrieve-a-receipt).
```

Comment:
Unterschied zu Input line in anderen Tabs? Ansonsten wie in anderen Tabs nur der Verweis...